CHANGELOG.md

```diff
@@ -2,6 +2,50 @@
 
 <!-- next version -->
 
+## 0.104.0
+
+### 🛑 Breaking changes 🛑
+
+- `opamp`: Adds support for v1beta1 OpenTelemetry Collector API in the OpAMP Bridge (#2985)
+  This change adds support for the OpAMP Bridge to manage and apply OpenTelemetry Collectors using the v1beta1 API in 
+  the OpAMP Bridge. This change removes support for applying OpenTelemetry Collectors using the v1alpha1 API version.
+  The v1beta1 API is the latest version of the OpenTelemetry Collector API and is the recommended version for new 
+  deployments.
+
+### 💡 Enhancements 💡
+
+- `operator`: Disables `component.UseLocalHostAsDefaultHost` by default (#3139)
```

Comment:
```suggestion
- `collector`: Since collector version `0.104.0` the collector listens on `localhost` instead of `0.0.0.0` by default. To avoid breaking changes the the `component.UseLocalHostAsDefaultHost` feature-gate is disabled by the Operator.  (#3139)
```

cc @pavolloffay

Reply:
double `the` :)

Reply:
I think the #3139  is a different issue